content/2-4.md

```diff
@@ -1,43 +1,43 @@
 ---
-title: "[Draft] Module 4: Tabular Information"
-permalink: /curricula/developing-accessible-content/tabular-information/
-ref: /curricula/developing-accessible-content/tabular-information/
+title: "[Draft] Module 4: Tables"
+permalink: /curricula/developer-modules/tables/
+ref: /curricula/developer-modules/tables/
 lang: en
 github:
   repository: w3c/wai-curricula
   path: content/2-4.md
 license: creative-commons
+acknowledgements: /curricula/acknowledgements/
 footer: >
   <p><strong>Date:</strong> $Date</p>
-  <p><strong>Editors: </strong>Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. </p>
+  <p><strong>Editors:</strong> Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. ACKNOWLEDGEMENTS lists contributors and credits.</p>
   <p>Developed by the Education and Outreach Working Group (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). Developed with support from the <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide Project</a> funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 822245).</p>
 navigation:
-  previous: /curricula/developing-accessible-content/images-and-graphics/
-  next: /curricula/developing-accessible-content/forms-and-input-elements/
+  previous: /curricula/developer-modules/images/
+  next: /curricula/developer-modules/forms/
 ---
 
 ## Introduction
 {:.no-display}
 
-Courses based on this module:
+Courses based on this module should:
 
-* explain use of proper coding to associate header and data cells and to describe the topic and purpose of a table
-* demonstrate how people with disabilities use table structure and markup of header and data cells to navigate and associate information in tables
+* demonstrate how people with disabilities use table structures, including headers and data cells, to understand information presented in tables
+* explain coding techniques to mark up headers and data cells, and to describe the topic and purpose of accessible tables
```

Comment:
"purpose of tables" rather than "purpose of accessible tables"?

Reply:
Makes sense. Will change.